src/constants/pools.ts

```diff
@@ -81,6 +81,7 @@ export type Pools = {
   Metadata: Record<string, PoolMetadata>;
   DisabledJoins: string[];
   BrandedRedirect?: Record<string, string>;
+  Deprecated?: string[];
```

Comment:
Could you change this schema to:
```suggestion
  Deprecated?: Record<string, DeprecatedDetails>;
```
Where DeprecatedDetails is:
```
type DeprecatedDetails = {
    newPool?: string;
    suggestedPools?: string[];
}
```
Once you've done this I'll get you a list of new pools to point these deprecated pools too. Pon is working on some designs to include a link to the new or suggestedPools on the pool page.

Reply:
done